_sass/components/_nav.scss

```diff
@@ -15,6 +15,13 @@
   }
 }
 
+@include at-media('desktop') {
+  .usa-header--extended .usa-navbar {
+    max-width: 100%;
+    justify-content: space-between;
+  }
+}
```

Comment:
To avoid any potential confusion with expected design system behavior for default components, I might suggest adding a custom class for the additional customizations.

Or perhaps using utility classes like [`maxw-full`](https://designsystem.digital.gov/utilities/height-and-width/#maxw) [`flex-justify`](https://designsystem.digital.gov/utilities/flex/#utility-flex-justify) which, if added to the partner header, would have the added advantage of not impacting the base layout.

Reply:
Good point, thanks @aduth  !

Reply:
I just realized those utility classes don't support responsive prefixes like `desktop:` which may be a problem. I think we could configure the design system to make those available though, if you wanted to go that route.

Reply:
Yes I was noticing that issue too, I did see one other place someone used !important in a class to overwrite so I did that but maybe better to configure it, if it's not too heavy of a lift.